commands/Fun/say.js

```diff
@@ -0,0 +1,19 @@
+const { Command } = require("klasa");
+
+class Say extends Command {
```

Comment:
This doesn't really respect the formst we use in the other pieces. Please use `module.exports = class extends Command`